src/Tribe/Main.php

```diff
@@ -118,12 +118,20 @@ public function load_assets() {
 				array( 'tribe-dependency', 'dependency.js', array( 'jquery', 'underscore' ) ),
 				array( 'tribe-dependency-style', 'dependency.css' ),
 				array( 'tribe-notice-dismiss', 'notice-dismiss.js' ),
+				array( 'tribe-common', 'tribe-common.js' ),
 			),
 			'admin_enqueue_scripts',
 			array(
 				'filter' => array( Tribe__Admin__Helpers::instance(), 'is_post_type_screen' ),
+				'localize' => (object) array(
+					'name' => 'system_info',
```

Comment:
Now that I've looked through this twice - I feel stupid for missing this the first time through. I think we should namespace this variable to `tribe_system_info` to avoid conflicts with other plugins/themes.